src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -18,6 +18,14 @@
           <dd i18n>{{ detailsCardData.orchestrator || 'Orchestrator is not available' }}</dd>
           <dt>Ceph version</dt>
           <dd>{{ detailsCardData.cephVersion }}</dd>
+          <dd *ngIf="upgradeInfo$ | async as info"
+              i18n>
+            <a *ngIf="info.versions.length > 0"
+               (click)="upgradeModal()"
+               href="#/upgrade">
+              Upgrade available
+            </a>
+          </dd>
```

Comment:
do we need to go to `#/upgrade`? Ideally we should show a progress right next to version to indicate an upgrade in progress but we can take that as in RFE. WDYT?

Btw, I saw too much space in between version and Upgrade message so we can remove the margin-bottom for the version field. and also I added an icon to see how it looks,
![Screenshot from 2024-03-22 12-36-20](https://github.com/ceph/ceph/assets/71764184/71a35b8e-1554-4a9f-b300-27466dca6ae3)


```suggestion
          <dd class="mb-0">{{ detailsCardData.cephVersion }}</dd>
          <dd *ngIf="upgradeInfo$ | async as info"
              i18n>
            <a *ngIf="info.versions.length > 0"
               (click)="upgradeModal()">
               <i [ngClass]="icons.up"></i>
              Upgrade available
            </a>
          </dd>
```

Reply:
> do we need to go to #/upgrade?

I thought that if you wanted to upgrade, being in the main page to upgrade would be the ideal, since there should lay all you might need to complete and monitor the upgrade.

>Ideally we should show a progress right next to version to indicate an upgrade in progress but we can take that as in RFE. WDYT?

Sure, we can take it later on, I wasn't aware of that

>Btw, I saw too much space in between version and Upgrade message so we can remove the margin-bottom for the version field. and also I added an icon to see how it looks

Looks good, I'll make the changes, thanks

Reply:
> main page to upgrade would be the ideal

maybe we just need to redirect to `#/upgrade/progress` once the upgrade is started from the modal rather than taking the user to upgrade component and opening the modal form. But I am okay with your current approach for the time being..

Reply:
I thought maybe we can have a modal open on the overview dashboard itself and start from there. 

If the upgrade in progress, we take change the icon to progress and let user go on upgrade page and navigate to progress.